libc/test/src/stdlib/SortingTest.h

```diff
@@ -7,19 +7,21 @@
 //===----------------------------------------------------------------------===//
 
 #include "src/__support/macros/config.h"
-#include "src/stdlib/qsort_data.h"
+#include "src/stdlib/qsort.h"
 #include "test/UnitTest/Test.h"
 
+#include <iostream>
+
```

Comment:
We can't include libc++ headers, so this needs to be removed.

Reply:
Good catch, that's debug leftover.